featurebyte/query_graph/node/schema.py

```diff
@@ -53,6 +54,30 @@ def create_feast_data_source(
         """
         raise NotImplementedError()
 
+    def get_offline_store_config(self, user_name: Optional[str], password: Optional[str]) -> Any:
```

Comment:
Would it make sense for the function signature to accept a `BaseDatabaseCredential` to support other credential types in the future?